### PR TITLE
Added Travic CI i.e. .travis.yml

### PR DESCRIPTION
**Pull request by vaibhavk**
Fixes #28

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: go
+
+go:
+  - "1.x"
+
+script: go test -v
```
[the remark this answers is not in the document]
Comment (vaibhavk) [reply .travis.yml]: Adding `go build` would be sufficient for the same?

[the remark this answers is not in the document]
Comment (hellozee) [reply .travis.yml]: Yup

**Comment by farhaanbukhsh** [comment]
I am just wondering guys can we also add some basic test here. This can have an impact on the coming people who wants to write tests.

**Comment by farhaanbukhsh** [comment]
should we add a linter as well?

**Comment by hellozee** [comment]
@farhaanbukhsh what about this https://fixmie.com/

**Comment by farhaanbukhsh** [comment]
Hey @vaibhavk  is this pr ready?

**Comment by hellozee** [comment]
Thanks, merging it

**Comment by vaibhavk** [comment]
Thanks for the guidance all the way through @hellozee

**Pull request merged**